priv/templates/slime/phx.gen.html/controller.ex

```diff
@@ -0,0 +1,68 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>Controller do
```

Comment:
Do we need to duplicate the non-view files?  The controller and controller test files don't differ between applications using EEX vs Slime templates.  I think we can DRY this up a bit and only need to maintain a single version of the non-view related files that don't vary based on the given `template_format` option.